test/katex-spec.js

```diff
@@ -17,6 +17,7 @@ const defaultSettings = new Settings({});
 const defaultOptions = new Options({
     style: Style.TEXT,
     size: 5,
+    maxSize: Infinity,
```

Comment:
nice

Reply:
hmm... this looks like a default, it should probably go inside the `Options` constructor.

Reply:
There would be a bit of code duplication if we added this to the `Options` constructor because the default is already being parsed in the `Settings` constructor:

https://github.com/Khan/KaTeX/blob/ccf09786cc747080cdaa091c2ba2251861d1d138/src/Settings.js#L27

Is that okay, or is there a more elegant solution?

Reply:
That's too bad about the duplication.  Let's leave this as is for now.